mypy.ini

```diff
@@ -52,3 +48,15 @@ ignore_missing_imports = True
 
 [mypy-signedjson.*]
 ignore_missing_imports = True
+
+[mypy-prometheus_client.*]
```

Comment:
because we don't have type stubs for these modules, so therefore we don't want to typecheck them. We don't want to ignore missing imports completely, just on dependencies without type hints.

Reply:
ok, but it just doesn't seem related to the subject of the PR and the text of the changelog.

I guess it's necesary because of https://github.com/matrix-org/synapse/pull/6137/files#diff-b91f3d5bd63fcd17221b267e851608e8R169?